### PR TITLE
HybridDataManager - adding API version 2019-06-01 and examples

### PR DESCRIPTION
**Pull request by prtank**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by markcowl** [comment]
@prtank  per the checklist above, this PR needs arm feedback.  Adding appropriate label

**Review by majastrz — APPROVED** [review]
Diffing with 2016-06-01 shows only a small change. When adding a new API, please make the first commit an unmodified copy of the previous API version at the folder of the new API version. This makes reviews very easy because then we simply diff first commit with last and get the actual delta.

Otherwise, changes look good from ARM side. Signing off...

File: specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2019-06-01/hybriddata.json [hunk 1]
```diff
@@ -0,0 +1,2654 @@
...
+          {
+            "$ref": "#/parameters/dataManagerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The data store/repository object.",
```
Comment (markcowl) [inline specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2019-06-01/hybriddata.json:1201]: Completed request for Create/Update.

File: specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2019-06-01/hybriddata.json [hunk 2]
```diff
@@ -0,0 +1,2654 @@
...
+            "name": "$filter",
+            "in": "query",
+            "description": "OData Filter options",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of job definitions in that resource.OK",
```
Comment (markcowl) [inline specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2019-06-01/hybriddata.json:1405]:
```suggestion
            "description": "The list of job definitions in the resource.",
```

**Review by markcowl — CHANGES_REQUESTED** [review]
A few minor changes in comments, otherwise, LGTM

**Review by markcowl — APPROVED** [review]

**Pull request merged**